TAF/testScenarios/integrationTest/UC_kuiper/export_by_kuiper_rules.robot

```diff
@@ -25,9 +25,9 @@ Kuiper001 - Add a new rule and export to MQTT
     Given Set Test Variable  ${device_name}  kuiper-mqtt-device
     And Set Test Variable  ${command}  ${PREFIX}_GenerateDeviceValue_INT8_RW
     And Set Test Variable  ${resource}  ${PREFIX}_DeviceValue_INT8_RW
-    And Start process  python ${WORK_DIR}/TAF/utils/src/setup/mqtt-subscriber.py rules-events-kuiper ${resource} arg &   # Process for MQTT Subscriber
+    And Start process  python ${WORK_DIR}/TAF/utils/src/setup/mqtt-subscriber.py rules-events-kuiper ${resource} ${EX_BROKER_PORT} false arg &   # Process for MQTT Subscriber
```

Comment:
Is this expecting a rule action in eKuiper to export data to the external broker?

Reply:
Yes, this is expecting to export data to the external broker. Would you expect the action to export data to the internal broker?

Reply:
Nope, just clarifying. :-)